### ISSUE TITLE
Load DDS files in DevIlImageLoader without decompression

### ISSUE THREAD

**Issue by mosra** [mosra/magnum-plugins#27]
I thought this would be just a few lines changed, but apparently not and the more I'm trying to search anything about this in the incredibly sparse docs the more miserable I get.

What I found out so far:

- I thought that `IL_IMAGE_FORMAT` would be set to something DXT-related for `*.dds`, but it's set to RGBA for a DXT5 file.
- So I thought that this is in `IL_IMAGE_TYPE`. That returns 0x1401 for *all* images and that value is not in the header. What's that?!
- The IL header has some defines for DXT like `IL_DXTC_FORMAT`, `IL_KEEP_DXTC_DATA` or `IL_DXT5`, but I failed miserably at searching for their use when loading images.

So far it seems that there is no way to load the DDS files without decompression and that would be very bad, because what's the point of doing that? A plain JPEG would match such use case much better.

Cc: @Alicemargatroid

**Comment by Alicemargatroid** [comment]
@mosra When you load it without converting it, you can read it out in raw format I think.
I'll look into it, I've got some time this weekend, but no guarantees.

**Comment by mosra** [comment]
Thank you! :)

**Comment by alicemargatroid** [comment]
@mosra Am blocked by a new issue I've raised, which I've spent four weekends attempting to fix!
